package.json

```
@@ -55,7 +55,7 @@
     "tap": "^16.0.1"
   },
   "engines": {
-    "node": "^14.17.0 || ^16.13.0 || >=18.0.0"
+    "node": "^16.13.0 || >=18.0.0"
```

```suggestion
    "node": "^16.14.0 || >=18.0.0"
```

This is what our other repos are at.

---

Good catch!

---

Fixed in https://github.com/npm/ssri/pull/137/commits/9d54482bee5c83e2b16f75444c8f3324aa2b771e